docs/xls-66d-lending-protocol/reference/transactions/loanset.md

```diff
@@ -0,0 +1,108 @@
+---
+seo:
+    description: Creates a new `Loan` ledger entry to represent a loan agreement between a Loan Broker and Borrower.
+labels:
+  - Transactions
+  - Lending Protocol
+---
+# LoanSet
+[[Source]](https://github.com/ "Source")
+
+Creates a new `Loan` ledger entry, representing a loan agreement between a _Loan Broker_ and _Borrower_.
+
+The `LoanSet` transaction is a mutual agreement between the _Loan Broker_ and _Borrower_, and must be signe by both parties. The multi-signature flow is as follows:
+
+1. The borrower creates a new transaction with the preagreed terms of the loan and signs the transaction.
+2. The lender signs over all signing fields, including the signature of the borrower.
+
+_(Requires the [Lending Protocol amendment][] {% not-enabled /%})_
+
+## Example LoanSet JSON
+
+**TODO: Add real example.**
+```json
+{
+  "TransactionType": "LoanSet",
+  "Account": "rEXAMPLE9AbCdEfGhIjKlMnOpQrStUvWxYz",
+  "Fee": "12",
+  "Flags": 0,
+  "LastLedgerSequence": 7108682,
+  "Sequence": 8,
+  "LoanBrokerID": "ABCDEF1234567890ABCDEF1234567890ABCDEF1234567890ABCDEF1234567890",
+  "Borrower": "rBORROWER9AbCdEfGhIjKlMnOpQrStUvWxYz",
+  "Data": "5468697320697320617262697472617279206D657461646174612061626F757420746865206C6F616E2E",
+  "LoanOriginationFee": 100,
+  "LoanServiceFee": 10,
+  "LatePaymentFee": 5,
+  "FullPaymentFee": 20,
+  "InterestRate": 500,
+  "LateInterestRate": 1000,
+  "ClosingInterestRate": 200,
+  "PrincipalRequested": 10000,
+  "StartDate": 1234567890,
+  "PaymentsTotal": 12,
+  "PaymentInterval": 2592000,
+  "GracePeriod": 604800,
+  "Lender": {
+    "SigningPubKey": "...",
+    "Signature": "..."
+  }
+}
+```
+
+## LoanSet Fields
+
+In addition to the [common fields][], `LoanSet` transactions use the following fields:
+
+| Field Name           | JSON Type | Internal Type | Required? | Description |
```

Comment:
Looks like some of these fields have been updated:

https://github.com/XRPLF/XRPL-Standards/blob/81e2ff8954763fa572ba0bb937333391327dca63/XLS-0066d-lending-protocol/README.md#321-loanset-transaction